question-generation/backend.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+from transformers import BartTokenizer, BartForConditionalGeneration
+from pathlib import Path
+import random
+import torch
+
+
+def load_contexts_from(file_name: Path) -> "list[str]":
```

Comment:
Not sure this is worth a function, when it's just `Path(filename).read_text().split('\n')`.

Reply:
Removing text input operation as it is not even relevant anymore.